iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/TerminalRefXml.java

```diff
@@ -18,21 +19,34 @@
  */
 public final class TerminalRefXml {
 
-    public static void writeTerminalRef(Terminal t, NetworkXmlWriterContext context, String elementName) throws XMLStreamException {
-        writeTerminalRef(t, context, context.getVersion().getNamespaceURI(), elementName);
+    public static void writeTerminalRef(Terminal t, NetworkXmlWriterContext context, String elementName, String id) throws XMLStreamException {
+        writeTerminalRef(t, context, context.getVersion().getNamespaceURI(), elementName, id);
     }
 
-    public static void writeTerminalRef(Terminal t, NetworkXmlWriterContext context, String namespace, String elementName) throws XMLStreamException {
-        writeTerminalRef(t, context, namespace, elementName, context.getWriter());
+    public static void writeTerminalRef(Terminal t, NetworkXmlWriterContext context, String namespace, String elementName, String id) throws XMLStreamException {
+        writeTerminalRef(t, context, namespace, elementName, id, context.getWriter());
     }
 
     public static void writeTerminalRef(Terminal t, NetworkXmlWriterContext context, String namespace, String elementName, XMLStreamWriter writer) throws XMLStreamException {
+        writeTerminalRef(t, context, namespace, elementName, null, writer);
+    }
+
+    public static void writeTerminalRef(Terminal t, NetworkXmlWriterContext context, String namespace, String elementName, String id, XMLStreamWriter writer) throws XMLStreamException {
```

Comment:
From my point of view, the read/write method should take the Identifiable object instead of an idea:
```
 public static void writeTerminalRef(Connectable owner, Terminal t, NetworkXmlWriterContext context, String namespace, String elementName, XMLStreamWriter writer) throws XMLStreamException;
```